src/goto-programs/interpreter.cpp

```diff
@@ -1181,6 +1204,9 @@ Function: interpretert::get_size
 
 size_t interpretert::get_size(const typet &type)
 {
+  if(unbounded_size(type))
+    return 2ULL << 32ULL;
```

Comment:
I added comments about this in this commit https://github.com/diffblue/cbmc/pull/689/commits/1a97a94c08702967ee889dc788dd4cf087d0f2db